sync/clients/databricks.py

```diff
@@ -92,9 +96,9 @@ def get_run(self, run_id: str) -> dict:
         )
 
     def _send(self, request: httpx.Request) -> dict:
-        response = self._client.send(request)
+        response = self._send_request(request)
 
-        if response.status_code >= 200 and response.status_code < 300:
+        if 200 <= response.status_code < 300:
```

Comment:
PyCharm suggested this little change to me

Reply:
Nice. Didn't know you could do that. Doesn't make a lot of sense to me from an operator associativity standpoint...

Reply:
https://docs.python.org/3/reference/expressions.html#comparisons